### PR TITLE
Update changelog for 0.22.0-insiders3.

### PR DESCRIPTION
**Pull request by sean-mcmanus**
@lh123 Do you have a "real name" you want us to use in our changelog for your contribution?

### CODE REVIEW

**Comment by sean-mcmanus** [comment]
Let me know if I forgot a change you want me to add.

There may be additional fixed isues, but I'll have to ask users for confirmation after we ship.

**Comment by sean-mcmanus** [comment]
We can update the changelog for 0.22.0 if anyone wants.

**Comment by lh123** [comment]
Thank you for your notice, using `lh123` is enough.

File: Extension/CHANGELOG.md [hunk 1]
```diff
@@ -1,13 +1,27 @@
...
+* Fix bugs with squiggles and IntelliSense updating after edits. [#1779](https://github.com/Microsoft/vscode-cpptools/issues/1779), [#3124](https://github.com/Microsoft/vscode-cpptools/issues/3124), [#3260](https://github.com/Microsoft/vscode-cpptools/issues/3260)
+* Fix formatting (and other non-IntelliSense operations) being blocked by IntelliSense processing. [#1928](https://github.com/Microsoft/vscode-cpptools/issues/1928)
+* Show a prompt for changing `C_Cpp.updateChannel` to `Insiders`. [#3089](https://github.com/Microsoft/vscode-cpptools/issues/3089)
+  * lh123 (@lh123) [PR #3221](https://github.com/Microsoft/vscode-cpptools/pull/3221)
+* Fix bugs with the Insider reload prompt appearing when it shouldn't. [#3206](https://github.com/Microsoft/vscode-cpptools/issues/3206)
+* Enable `F5` and the `Build and Debug Active File` context menu command. [PR #3244](https://github.com/Microsoft/vscode-cpptools/pull/3244)
+* Fix IntelliSense process crashing with Mac Sierra 10.12 or earlier. [#3256](https://github.com/Microsoft/vscode-cpptools/issues/3256)
+* Fix incorrect `c_cpp_properties.json` squiggles with `cl.exe`. [#3263](https://github.com/Microsoft/vscode-cpptools/pull/3263)
```
Comment (sean-mcmanus) [inline Extension/CHANGELOG.md:11]: The PR is missing in this link -- I'll fix it for the next changelog.

**Pull request merged**